lib/interfaces/entity.type.ts

```diff
@@ -0,0 +1,3 @@
+import { EntitySchema } from 'typeorm';
+
+export type Entity = Function | EntitySchema;
```

Comment:
Can we rename it to, for example, `EntityClassOrSchema` instead? To avoid collision with the `Entity` from `typeorm`.